crates/red_knot_python_semantic/src/semantic_index/use_def/symbol_state.rs

```diff
@@ -313,7 +313,7 @@ impl SymbolBindings {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, salsa::Update)]
```

Comment:
deriving `salsa::Update` was required here or the compiler started giving me odd lifetime complaints due to the new `bindings_at_star_imports: FxHashMap<Definition<'db>, SymbolState>` fields on `UseDefMap` and `UseDefMapBuilder`